interfaces/builtin/common.go

```diff
@@ -87,6 +87,12 @@ func (iface *commonInterface) SanitizeSlot(slot *interfaces.Slot) error {
 	return nil
 }
 
+// SanitizePlug checks and possibly modifies a plug.
```

Comment:
Why do you add an empty sanitize helper if those are optional?

Reply:
It is here to make things future proof, but your question is valid and YAGNI etc - so I think its fine to remove it as long as we remember that if we add SanitizePlug to commonInterface we need to look over the existing commonInterface embedders.